### PR TITLE
feat: add cd onboarding path

### PR DESCRIPTION
**Pull request by floreks**
## Summary

Ref: https://app.asana.com/0/1204017270640279/1205710362178675/f

## Test Plan

Locally

## Checklist

- [ ] If required, I have updated the Plural documentation accordingly.
- [ ] I have added tests to cover my changes.
- [ ] I have added a meaningful title and summary to convey the impact of this PR to a user.
- [ ] I have added relevant labels to this PR to help with categorization for release notes.

### CODE REVIEW

**Review by michaeljguarino — APPROVED** [review]

File: www/src/components/shell/onboarding/sections/overview/OverviewStep.tsx [hunk 1]
```diff
@@ -45,12 +57,27 @@ function OverviewStep({ onNext }) {
...
 
-          <Span>1. Configure your cloud and git credentials.</Span>
-          <Span>2. Configure your cluster’s workspace.</Span>
-          <Span>
-            3. Create your cloud shell where you can install applications. (25
-            minutes deploy wait time)
-          </Span>
+          {path === OnboardingPath.CD && (
+            <>
+              <Span>1. Configure your cloud and git credentials.</Span>
```
Comment (dogmar) [inline www/src/components/shell/onboarding/sections/overview/OverviewStep.tsx:62]:
```suggestion
              <Span>1. Configure your cloud and Git credentials.</Span>
```

File: www/src/components/shell/onboarding/sections/overview/OverviewStep.tsx [hunk 2]
```diff
@@ -45,12 +57,27 @@ function OverviewStep({ onNext }) {
...
+              <Span>
+                3. Create your cloud shell to deploy your Plural Console. (25m
+                minutes deploy wait time)
+              </Span>
+            </>
+          )}
+
+          {path === OnboardingPath.OSS && (
+            <>
+              <Span>1. Configure your cloud and git credentials.</Span>
```
Comment (dogmar) [inline www/src/components/shell/onboarding/sections/overview/OverviewStep.tsx:73]:
```suggestion
              <Span>1. Configure your cloud and Git credentials.</Span>
```

File: www/src/components/shell/onboarding/sections/overview/OverviewStep.tsx [hunk 3]
```diff
@@ -45,12 +57,27 @@ function OverviewStep({ onNext }) {
...
-            3. Create your cloud shell where you can install applications. (25
-            minutes deploy wait time)
-          </Span>
+          {path === OnboardingPath.CD && (
+            <>
+              <Span>1. Configure your cloud and git credentials.</Span>
+              <Span>2. Configure your Management Cluster's workspace.</Span>
+              <Span>
+                3. Create your cloud shell to deploy your Plural Console. (25m
+                minutes deploy wait time)
```
Comment (dogmar) [inline www/src/components/shell/onboarding/sections/overview/OverviewStep.tsx:66]:
```suggestion
                3. Create your cloud shell to deploy your Plural Console. (25
                minute deploy wait time)
```

File: www/src/components/shell/onboarding/sections/overview/OverviewStep.tsx [hunk 4]
```diff
@@ -45,12 +57,27 @@ function OverviewStep({ onNext }) {
...
+            </>
+          )}
+
+          {path === OnboardingPath.OSS && (
+            <>
+              <Span>1. Configure your cloud and git credentials.</Span>
+              <Span>2. Configure your cluster’s workspace.</Span>
+              <Span>
+                3. Create your cloud shell where you can install applications.
+                (25 minutes deploy wait time)
```
Comment (dogmar) [inline www/src/components/shell/onboarding/sections/overview/OverviewStep.tsx:77]:
```suggestion
                (25 minute deploy wait time)
```

File: www/src/components/shell/onboarding/sections/welcome/PathOption.tsx [hunk 5]
```diff
@@ -0,0 +1,51 @@
...
+
+import OnboardingCardButton from '../../OnboardingCardButton'
+
+function PathOption({
+  icon,
+  header,
+  description,
+  selected,
+  disabled,
+  tooltip,
```
Comment (dogmar) [inline www/src/components/shell/onboarding/sections/welcome/PathOption.tsx:11]: Shoudl remove as suggested by `GitHubActions / Lint`

File: www/src/components/shell/onboarding/sections/welcome/WelcomeStep.tsx [hunk 6]
```diff
@@ -0,0 +1,114 @@
...
+            selected={path === OnboardingPath.CD}
+            onClick={() => setPath(OnboardingPath.CD)}
+            icon={
+              <CloudIcon
+                size={40}
+                color="text-light"
+              />
+            }
+            header="Deploy your own applications"
+            description="Use Plural Continuous Deployments to  deploy applications and services from your organization."
```
Comment (dogmar) [inline www/src/components/shell/onboarding/sections/welcome/WelcomeStep.tsx:46]:
```suggestion
            description="Use Plural Continuous Deployments to deploy applications and services from your organization."
```

**Review by dogmar — CHANGES_REQUESTED** [review]
Just some minor grammar/lint suggestions. Otherwise, looks great!

**Review by dogmar — APPROVED** [review]

**Pull request merged**